editor/plugins/path_3d_editor_plugin.cpp

```diff
@@ -281,8 +281,15 @@ void Path3DGizmo::redraw() {
 	Ref<StandardMaterial3D> path_tilt_material = gizmo_plugin->get_material("path_tilt_material", this);
 	Ref<StandardMaterial3D> path_tilt_muted_material = gizmo_plugin->get_material("path_tilt_muted_material", this);
 	Ref<StandardMaterial3D> handles_material = gizmo_plugin->get_material("handles");
+	Ref<StandardMaterial3D> first_pt_handle_material = gizmo_plugin->get_material("first_pt_handle");
+	Ref<StandardMaterial3D> last_pt_handle_material = gizmo_plugin->get_material("last_pt_handle");
+	Ref<StandardMaterial3D> closed_pt_handle_material = gizmo_plugin->get_material("closed_pt_handle");
 	Ref<StandardMaterial3D> sec_handles_material = gizmo_plugin->get_material("sec_handles");
 
+	first_pt_handle_material->set_albedo(Color(0.0, 1.0, 0.0));
+	last_pt_handle_material->set_albedo(Color(1.0, 0.0, 0.0));
+	closed_pt_handle_material->set_albedo(Color(1.0, 1.0, 0.0));
```

Comment:
I find the pure yellow a bit hard to distinguish from white, maybe try a slightly orange color for the closed handle:

```suggestion
	first_pt_handle_material->set_albedo(Color(0.2, 1.0, 0.0));
	last_pt_handle_material->set_albedo(Color(1.0, 0.2, 0.0));
	closed_pt_handle_material->set_albedo(Color(1.0, 0.8, 0.0));
```

This also includes tweaks to the first/last point colors to look more visually pleasing.

![image](https://github.com/godotengine/godot/assets/180032/4a62cea4-fbe9-45aa-acd9-d1cfcf6f1ea5)

![image](https://github.com/godotengine/godot/assets/180032/b9571b69-b174-4a80-b1e8-a9b6bf9fa52a)